src/views/tutorial-view/index.tsx

```diff
@@ -60,13 +72,17 @@ export type CollectionContext = {
 }
 
 export type TutorialSidebarSidecarProps = Required<
-  Pick<SidebarSidecarLayoutProps, 'children' | 'headings' | 'breadcrumbLinks'>
+  Pick<
+    SidebarSidecarLayoutProps,
+    'children' | 'headings' | 'breadcrumbLinks'
+  > & { sidebarSections: CollectionCategorySidebarSection[] }
 >
 
 export default function TutorialView({
   layout,
   tutorial,
 }: TutorialViewProps): React.ReactElement {
+  const currentProduct = useCurrentProduct()
```

Comment:
Again `product` is already passed as a prop to this view. I feel like we should just use that.